api/resid/gvk_test.go

```diff
@@ -255,3 +255,35 @@ func TestSelectByGVK(t *testing.T) {
 		}
 	}
 }
+
+func TestIsNamespaceableKind(t *testing.T) {
+	var testCases = []struct {
```

Comment:
We should name each test case and use golang subtests for running each of them. There is an example here: https://github.com/kubernetes-sigs/kustomize/blob/719532e4df44f553fcfa5b51fb519eca2bcf4e9d/kyaml/openapi/openapi_test.go#L317

Reply:
okay, I added a name field for each test case and changed it to use t.Run like the example